### PR TITLE
create scope-validation-announcement.md

### PR DESCRIPTION
**Pull request by nmurav**

### CODE REVIEW

File: scope-validation-announcement.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# We are keep improving our service! But we might affect you , please read this.
+With the initial release of the ACS we allowed to request the scopes for the tokens. Please refer here for the [Access Token scopes](https://docs.microsoft.com/en-us/azure/communication-services/quickstarts/access-tokens?pivots=programming-language-javascript)
+
+However we have not enforced the VoIP and PSTN scopes. Meaning you could make a call with  a chat only scope in your token.
```
Comment (ansrin) [inline scope-validation-announcement.md:4]: maybe rephrase as "...we have not enforced the 'voip' scope for voip or pstn calls. Meaning you could make a voip call with 'chat' only scope or pstn call with 'pstn' only scope. "

File: scope-validation-announcement.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+Starting tomorrow we will start rolling out the update to enforce the scopes. That means if you have a token with chat only scope you will not be able to make a call.
```
Comment (ansrin) [inline scope-validation-announcement.md:6]: "...update to enforce the 'voip' scope."

Comment (ansrin) [inline scope-validation-announcement.md:6]: "...That means if you have a token with 'chat' only scope or 'pstn' only scope, you will not be able to make a voip call.

File: scope-validation-announcement.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+
+We will watch the telemetry to see affected customers and will be reaching out to customers who might experience the issues. However, we highly encourage you to review the tokens you have and if you need to have ability to make PSTN or VoIP calls, please ensure your token has the VoIP scope.
```
Comment (ansrin) [inline scope-validation-announcement.md:8]: do we need to mention "...watch the telemetry.." here ?
can we simply say  "We will be reaching out to customers who might experience call failures due to this 'voip' scope enforcement. However..."

File: scope-validation-announcement.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
+# We are keep improving our service! But we might affect you , please read this.
+With the initial release of the ACS we allowed to request the scopes for the tokens. Please refer here for the [Access Token scopes](https://docs.microsoft.com/en-us/azure/communication-services/quickstarts/access-tokens?pivots=programming-language-javascript)
+
+However we have not enforced the VoIP and PSTN scopes. Meaning you could make a call with  a chat only scope in your token.
+
+Starting tomorrow we will start rolling out the update to enforce the scopes. That means if you have a token with chat only scope you will not be able to make a call.
+
+We will watch the telemetry to see affected customers and will be reaching out to customers who might experience the issues. However, we highly encourage you to review the tokens you have and if you need to have ability to make PSTN or VoIP calls, please ensure your token has the VoIP scope. 
+
+Please any urgent escalation send to acsfeedback@microsoft.com
```
Comment (ansrin) [inline scope-validation-announcement.md:10]: "For any urgent escalation, please send to..." or "Please direct any urgent escalation to ...." ?

File: scope-validation-announcement.md [hunk 5]
```diff
@@ -0,0 +1,10 @@
+# We are keep improving our service! But we might affect you , please read this.
+With the initial release of the ACS we allowed to request the scopes for the tokens. Please refer here for the [Access Token scopes](https://docs.microsoft.com/en-us/azure/communication-services/quickstarts/access-tokens?pivots=programming-language-javascript)
```
Comment (ansrin) [inline scope-validation-announcement.md:2]: nit: "With the initial release of the ACS, we allowed to request specific scopes for the user access tokens, to achieve a specific ACS functionality.."

File: scope-validation-announcement.md [hunk 6]
```diff
@@ -0,0 +1,10 @@
+# We are keep improving our service! But we might affect you , please read this.
```
Comment (akania) [inline scope-validation-announcement.md:1]: nit
"# We keep improving our service! But changes might affect you , please read this"

File: scope-validation-announcement.md (hunk 3 above)
Comment (akania) [inline scope-validation-announcement.md:8]: "we highly encourage you to review how you issue new tokens, and if you need to have ability to make PSTN or VoIP calls, please ensure you issue tokens with correct scopes"

**Comment by tophpalmer** [comment]
@nmurav I added a commit that resolves much of the SDE feedback

File: scope-validation-announcement.md [hunk 7]
```diff
@@ -0,0 +1,11 @@
+# Breaking Change - November 3, 2020
+
+### We are keep improving our service! But we might affect you , please read this.
+
+Currently developers are **allowed but not required** to request the scopes for the tokens. Please refer here for the [Access Token scopes](https://docs.microsoft.com/azure/communication-services/quickstarts/access-tokens?pivots=programming-language-javascript). However we have not enforced the VoIP and PSTN scopes. Meaning you could make a call with  a chat only scope in your token.
```
Comment (ansrin) [inline scope-validation-announcement.md:5]: add this to make it very clear ,"....Meaning you could make a voip call with 'chat' only scope or pstn call with 'pstn' only scope. "

File: scope-validation-announcement.md [hunk 8]
```diff
@@ -0,0 +1,11 @@
...
+
+Starting 9:00AM November 3, 2020, Seattle time, we will start deploying a service update to enforce scopes. That means if you have a token with chat only scope you will not be able to make a call.
```
Comment (ansrin) [inline scope-validation-announcement.md:7]: if we say "service update to enforce scopes" ... it kind of gives an impression that we are enforcing all scopes, including pstn, which is not the case...hence it is better to be very specific, "...update to enforce the presence of 'voip' scope for voip and pstn calls."

Comment (ansrin) [inline scope-validation-announcement.md:7]: "...That means if you have a token with 'chat' only scope or 'pstn' only scope, you will not be able to make a voip or pstn call. All voip and pstn calls should carry the mandatory 'voip' scope in token.

File: scope-validation-announcement.md [hunk 9]
```diff
@@ -0,0 +1,11 @@
...
+
+We will monitor telemetry to find affected customers and are reaching out to customers who might experience issues. However, we highly encourage everyone to pro-actively review your token management flows and ensure you are correctly requesting necessary scopes.
```
Comment (ansrin) [inline scope-validation-announcement.md:9]: I have mentioned this in prev commit as well ... do we need to mention "...monitor the telemetry.." here ?
can we simply say "We will be reaching out to customers who might experience call failures due to this 'voip' scope enforcement. However..."

File: scope-validation-announcement.md [hunk 10]
```diff
@@ -0,0 +1,11 @@
...
+
+Please any urgent escalation send to [acsfeedback@microsoft.com](mailto:acsfeedback@microsoft.com) or file an issue in this repo.
```
Comment (ansrin) [inline scope-validation-announcement.md:11]: ""For any urgent escalation, please send to..." or "Please direct any urgent escalation to ...." ?

File: scope-validation-announcement.md [hunk 11]
```diff
@@ -0,0 +1,11 @@
+# Breaking Change - November 3, 2020
+
+### We are keep improving our service! But we might affect you , please read this.
```
Comment (ansrin) [inline scope-validation-announcement.md:3]: nit
"# We keep improving our service! but changes might affect you , please read this"

**Comment by matthewrobertson** [comment]
I would expect this to be under "release notes"

File: scope-validation-announcement.md (hunk 2 above)
Comment (nmurav) [reply scope-validation-announcement.md]: added

Comment (nmurav) [reply scope-validation-announcement.md]: added

File: scope-validation-announcement.md (hunk 3 above)
Comment (nmurav) [reply scope-validation-announcement.md]: added

File: scope-validation-announcement.md (hunk 4 above)
Comment (nmurav) [reply scope-validation-announcement.md]: Modified

File: scope-validation-announcement.md [hunk 12]
```diff
@@ -1,6 +1,6 @@
 # Breaking Change - November 3, 2020
 
-### We are keep improving our service! But we might affect you , please read this.
+### We are keep improving our service! But we cahnges affect you , please read this.
```
Comment (RinaRish) [inline scope-validation-announcement.md]: typo

File: scope-validation-announcement.md [hunk 13]
```diff
@@ -1,8 +1,8 @@
 # Breaking Change - November 3, 2020
 
-### We are keep improving our service! But we cahnges affect you , please read this.
+### We are keep improving our service! But we changes affect you , please read this.
```
Comment (RinaRish) [inline scope-validation-announcement.md]: but "we changes" something is missing in this sentence

**Review by tophpalmer — APPROVED** [review]

**Comment by ProbablePrime** [comment]
What's the status here @tophpalmer @nmurav ? 

From the looks of things in the files changed tab there is the same document twice in two different folders, I would expect this to only be included once. 

Can we get it merged?

**Comment by ProbablePrime** [comment]
@nmurav Can this be closed, Chris said it was invalid now.

Thank you
